api/pom.xml

```diff
@@ -239,11 +239,11 @@
                 <version>2.8</version>
                 <configuration>
 		    <additionalparam>-Xdoclint:none</additionalparam>
-                    <doctitle>Concurrency Utilities for Java EE ${project.version} API Specification</doctitle>
+                    <doctitle>Jakarta Concurrency ${project.version} Specification</doctitle>
                     <bottom>
-<![CDATA[Copyright &#169; 1996-2013,
+<![CDATA[Copyright &#169; 2019,
     <a href="http://www.oracle.com">Oracle</a>
-    and/or its affiliates. All Rights Reserved.
+    and/or its affiliates.
```

Comment:
Why did you remove the first copyright date?  It should be "1996, 2019".

Also, do not remove the "All Rights Reserved."  Oracle legal insists that it be included.

Reply:
Done